src/components/KTooltip/KTooltip.vue

```diff
@@ -15,7 +15,7 @@
 
     <template #content>
       <div
-        :id="tooltipId"
+        :id="tooltipId || uuidv4()"
```

Comment:
suggestion: can you move this to a computed property?

Reply:
No need making it computed since it's static - a constant will do. Fixing here: https://github.com/Kong/kongponents/pull/2081

Reply:
I was saying this but yeah static how you have it is fine 
```
const var = computed((): string => props.tooltipId || randomTooltipId)
```